### ISSUE TITLE
Gestures: hand phases [no changes needed]

### ISSUE THREAD

**Issue by gregorydimitriadis** [DCS-LCSR/SignStream3#656]
Related back to main gesture enhancements https://github.com/DCS-LCSR/SignStream3/issues/648

The terminology of the gesture phases (associated elements) is subject to change as per specifications.

**Comment by gregorydimitriadis** [comment]
@cneidle Would this basically just encompass a renaming of the gesture phases? Or is there any intention to have the ability to rename the gesture phases in-app? (I think not the latter, just confirming this to be the case).

**Comment by cneidle** [comment]
Yes - just renaming some or all of the gesture phases, potentially, for this implementation.  No need to have the phases renamable in-app. thanks.

**Comment by douglas-motto-at-rutgers** [comment]
Need to establish the exact number and ordering of phases.

**Comment by cneidle** [comment]
Confirmed that no changes are needed relative to what was previously developed for the co-speech gesture prototype.
